source/sass/blocks/blocks-common/welcome.scss

```
@@ -0,0 +1,6 @@
+.welcome {
+  background-image: url("../img/decor-welcom-block.svg"), url("../img/image-welcom-block.png");
+  background-repeat: no-repeat;
+  background-position: 0% 98.5%, 0% 0%;
```

Когда пишем 0, то единицы измерения не указываем
```suggestion
  background-position: 0 98.5%, 0 0;
```